docker_custodian/docker_gc.py

```diff
@@ -46,16 +48,21 @@ def is_container_old(container, min_date):
     if state.get('Ghost'):
         return True
 
+    created_date = dateutil.parser.parse(container['Created'])
+
+    # Don't delete recently created containers
+    if max_date and created_date > max_date:
+        return False
+
     # Container was created, but never used
-    if (state.get('FinishedAt') == YEAR_ZERO and
-            dateutil.parser.parse(container['Created']) < min_date):
+    if state.get('FinishedAt') == YEAR_ZERO and created_date < min_date:
```

Comment:
The scenario you describe in the description should be handled by this line. When a container it created it has this zero date, and the same max age value is used.

Does this not work anymore? Is it just a bug?

Or is the desired behaviour to have a different value from the "max age" ?